>From a944a0c2b39597859660891b096d151542c3c279 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sat, 4 Jun 2016 21:39:41 +0900
Subject: [PATCH 2/2] modify ubuntu apt line

Probably users want to use main/universe, then restricted/multiverse
since restricted/multiverse is non-free repo.
---
 scripts/chdist.pl | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/scripts/chdist.pl b/scripts/chdist.pl
index c085ce3..47c6101 100755
--- a/scripts/chdist.pl
+++ b/scripts/chdist.pl
@@ -391,10 +391,12 @@ EOF
 #deb http://ftp.debian.org/debian/ unstable main contrib non-free
 #deb-src http://ftp.debian.org/debian/ unstable main contrib non-free
 
-#deb http://archive.ubuntu.com/ubuntu xenial main restricted
-#deb http://archive.ubuntu.com/ubuntu xenial universe multiverse
-#deb-src http://archive.ubuntu.com/ubuntu xenial main restricted
-#deb-src http://archive.ubuntu.com/ubuntu xenial universe multiverse
+
+#deb http://archive.ubuntu.com/ubuntu xenial main universe
+#deb-src http://archive.ubuntu.com/ubuntu xenial main universe
+
+#deb http://archive.ubuntu.com/ubuntu xenial restricted multiverse
+#deb-src http://archive.ubuntu.com/ubuntu xenial restricted multiverse
 EOF
     }
     close FH;
-- 
2.8.1

